tests/yast2_gui/yast2_software_management.pm

```diff
@@ -19,8 +19,13 @@ use testapi;
 sub run {
     my $self = shift;
     select_console 'x11';
-    $self->launch_yast2_module_x11('sw_single', match_timeout => 120);
-    send_key "alt-a";    # Accept => Exit
+    $self->launch_yast2_module_x11('sw_single', match_timeout => 25);
+    # Accept => Exit, or get to the installation report
+    send_key 'alt-a';
+    if (check_screen('sw_single_ui_installation_report', 10) && match_has_tag('sw_single_ui_installation_report')) {
```

Comment:
In case you wanted to use multi tag assert here, we could reuse `generic-desktop` needle.

Reply:
The needle is used in the ``post_run_hook``, therefore we would match the same needle twice. 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/y2x11test.pm#L67

Reply:
We would detect if `sw_single_ui_installation_report` is there or not, which gives us that flexibility not to do version checks. Point is that `check_screen('sw_single_ui_installation_report', 10) && match_has_tag('sw_single_ui_installation_report')` doesn't make any sense, as if check_screen matches, there is no need to check if the match has same tag as we tried to match (as it's same).
Other point would be to try spare 10 seconds in case screen is not shown, for that we could have used `generic-desktop`, which is asserted once more in `post_run_hook` as you've correctly stated. Hope it's more clear now =)